services/provider/server/consumer.go

```diff
@@ -17,16 +16,12 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-var (
-	errTicketAlreadyExists = errors.New("onboarding ticket already used by another storageConsumer")
-)
-
 type ocsConsumerManager struct {
-	client       client.Client
-	namespace    string
-	nameByTicket map[string]string
-	nameByUID    map[types.UID]string
-	mutex        sync.RWMutex
+	client         client.Client
+	namespace      string
+	nameByTicketID map[string]string
```

Comment:
Is there really a need for this map to exists after all of the changes? We had a need because we didn't had the consumer up and running yet

Reply:
removed.